### ISSUE TITLE
Blockchain API callbacks are reaching too late.

### ISSUE THREAD

**Issue by ahsan047** [blockchain/api-v1-client-php#79]
When i generate receiving address and send payment to it. Call backs are taking too much time to notify/arrive. Probably taking 2,3,4 and sometimes 5 hour to reach the first confimation(0 confirmation and then so on first second and third...) 
What is the problem , anyone here to explain plz..
note: this is my first post on github. don't know much about it... Thanks in anticipation

**Comment by irfankhan86004** [comment]
Hello All,

Greeting the day...!

I generated one wallet address and transfer payment on that wallet address, but my callback url called delay. for example, i transfer the amount on the 12:00 am, and my callback url called 4-5 hours delay.
i already generate the ticket on the support of the blockchain, but they people not provide the suffice answer of this issue,

I hope you guys give me the positive response.

**Comment by ahsan047** [comment]
same problem with me and not solved yet... please share here whenever you get it solve any time

**Comment by ahsan047** [comment]
@irfankhan86004 one more thing that when I notified with call back i response with echo "*ok*"; it still doesn't stop sending me notification. are you facing the same issue? if not please share you code if you can... mine code is here. https://bitcoin.stackexchange.com/questions/80425/how-to-stop-call-backs-from-blockchain-api-callback-service?noredirect=1#comment93490_80425

**Comment by irfankhan86004** [comment]
yes, you are very true, i applied that '*ok* ' even though they sending me the notification for the confirmation. and i generate the ticket for this callback issue, but they not provide the proper solution

**Comment by ahsan047** [comment]
no one is replying there too... :( please share your email if i find a solution i will share with you, and if you find it anytime don't forget to share with me at itsahsan047@gmail.com 👍

**Comment by irfankhan86004** [comment]
Yes sure

**Comment by ahsan047** [comment]
any solution yet?
